APIs/schemas/receiver-stage-schema.json

```diff
@@ -41,13 +41,34 @@
         }
       }
     },
+    "transport": {
```

Comment:
Is this intended to be read-only? If so I wonder whether it might be better off being in its own resource